scene/2d/remote_transform_2d.cpp

```diff
@@ -114,6 +114,14 @@ void RemoteTransform2D::_notification(int p_what) {
 			_update_cache();
 		} break;
 
+		case NOTIFICATION_RESET_PHYSICS_INTERPOLATION: {
+			if (cache.is_valid()) {
+				_update_remote();
+				Node2D *n = Object::cast_to<Node2D>(ObjectDB::get_instance(cache));
+				n->reset_physics_interpolation();
```

Comment:
It might be worth enclosing this:
```
if (n)
{
n->reset_physics_interpolation();
}
```
Just to eliminate possibility of crash if the node has been deleted.

Reply:
Ah yes I thought that the `cache.is_valid()` would ensure that but I see that _update_remote also do this check. I'll fix that later today